no-literal-securerandom-exact-dljc-cmd.sh

```diff
@@ -1,29 +1,58 @@
 #!/bin/sh
 
-## This is an example "one-command with no arguments" script that controls run-dljc.sh. It is copied directly
-## from the experiments for the Continuous Compliance paper. If you want to re-use this script, you'll have to change
-## different things depending on what you're doing differently:
-# * If you are running on a non-CSE machine, change the definitions of JAVA11_HOME and JAVA8_HOME.
-# * EVERYONE must change the definitions of CHECKERFRAMEWORK and ANDROID_HOME. Setting ANDROID_HOME is optional.
-# * EVERYONE must change all instances of "securerandom" in this file to the name of the .list file containing your target repositories.
-# * If you are running a checker other than the no-literal checker, change the -c, -q, -l, and -s arguments to run-dljc.sh to make sense
-#   for the checker you are running.
-# * Make sure that all the absolute paths in this file match your machine/experimental directory.
+## This script runs run-dljc.sh with appropriate arguments and
+## environment variables. It is copied from the experiments for the
+## Continuous Compliance paper. If you want to re-use this script,
+## you'll have to change environment variables and paths below.  Note
+## that this script is intended for use with a custom typechecker
+## (i.e., a typechecker that is not in the main Checker Framework
+## distribution).  If your typechecker is in the main Checker
+## Framework distribution, you should use run-dljc.sh or
+## configure-and-exec-dljc.sh directly rather than making a copy of
+## this script.
+
+## Change these if you are running on a non-CSE machine.
 
 export JAVA11_HOME=/usr/lib/jvm/java-11-openjdk/
 export JAVA8_HOME=/usr/lib/jvm/java-1.8.0-openjdk
+
+## Everyone must change these.
+
+export PARENTDIR=${HOME}/compliance-experiments/fse20
+export CHECKERFRAMEWORK=${PARENTDIR}/checker-framework
+checker=org.checkerframework.checker.noliteral.NoLiteralChecker
+checkername=no-literal
+repolist=securerandom.list
+
+# the stub files for the checker being used
```

Comment:
Why is this necessary?  I expect these particular stubs to be automatically used by the checker.  Are these extra stubs that are written by the experimenter, beyond those built into the checker?

Reply:
The no-literal checker doesn't load stubs automatically (nor do most of my custom checkers). Especially for compliance, the stubs in use are different depending on the context. So, I found it more convenient to specify the stubs here.